### DISCUSSION TITLE
Any submods to use?

### DISCUSSION THREAD

**Discussion by arixfyeilol** [Monika-After-Story/MonikaModDev#9906] [general - submod / external content]
_Any submods are nice, I really don't mind it. Thank you and sorry if it's a bother._

**Comment by TooMuchInvigilation** [comment]
Hi. I'd recommend Custom Rooms (especially Furnished Spaceroom and Den), Auto Atmos Change (changes the weather, so it matches the weather in your location) and Submod Updater Plugin.

**Reply by arixfyeilol** [reply]
_Thank you! But I already have these submods. Anyways, thank you very much!_
